packages/cli-repl/src/format-output.ts

```diff
@@ -71,6 +75,12 @@ function formatDatabases(output): string {
   return textTable(tableEntries, { align: ['l', 'r'] });
 }
 
+function formatStats(output): string {
+  return Object.keys(output).reduce((str, c) => {
+    return `${str}\n${c}\n${inspect(output[c])}\n---\n`;
```

Comment:
up to you if you want this suggestion:
```suggestion
    return `${str}\n${clr(c, ['bold', 'yellow'])}\n\n${inspect(output[c])}\n---\n`;
```

Reply:
Oh woah I didn't realize I could just press 'commit suggestion', that's a sick feature! I just added it by hand.